server/register.go

```diff
@@ -46,6 +47,17 @@ var (
 	}
 )
 
+// isValidEmail determines if a given email is of form "name@example.com".
+func isValidEmail(email string) bool {
+	addr, err := mail.ParseAddress(email)
+	if err != nil {
+		return false
+	}
+	// mail.ParseAddress will parse the first email if given a list of emails.
+	// Ensure that the given string contains exactly this single email.
```

Comment:
why not using `mail.ParseAddressList(list string)` to make sure this func can handle email(s) correctly?